CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Why this particular version of cmake?

Btw, this change is not reflected in the changelog.

Reply:
I choose this version because according to the issue #306, the user base of libpointmatcher were still on Ubuntu 16.04 almost two years ago. So, I assumed that nobody were using Ubuntu 12.04 anymore. Then, @simonpierredeschenes and I decided that it was probably safe enough to change it since cmake 2.8.12 is shipped with Ubuntu 14.04. That being said, if the majority of the users are using Ubuntu 16.04, I guess it would be safe to update the version to 3.5.1. But, I know that it will break the CMakeLists.txt and that would require some additional work to make it work again. Please, tell me if this makes sense, I can be wrong, I'm still learning that kind of stuff. 🙂

About the changelog, I guess it would be mentioned for the next release?

Reply:
> Btw, is there any official position on which version of Ubuntu we support?

Not really, I guess? But it would probably be a good idea to have one.

Reply:
Maybe this question is a bit out of scope. It makes sense to open a separate issue and resolve this thread.